continuous-integration/python3.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# This script runs a passing subset of tests under Python 3.
+
+python setup.py install
```

Comment:
How? don't they run a ubuntu distro? python would default to python2... or have they make the move already? :)